README.md

```diff
@@ -65,3 +74,54 @@ func main() {
     }
 }
 ```
+
+### Server
+
+The Serer extends the default golang HTTP Server by adding a requestID and logger middleware. By default it handles the OSSignals, and it has a default shutdown timeout of 10 seconds.
+
+This Server is intended to be used by all ONS digital publishing services that require to serve HTTP. The following example shows how to use the Server:
+
+#### Creation
+
+Assuming you have created a router with your API handlers, you can create the http server like so:
+
+```go
+import rchttp "github.com/ONSdigital/dp-net/http"
+    ...
+	httpServer := server.New(bindAddr, router)
+    httpServer.HandleOSSignals = false
```

Comment:
Indentation?

Reply:
I think one line uses tab and the other spaces - I will make it consistent